### PR TITLE
Import nirfmxbt metadata to grpc-device.

### PR DESCRIPTION
**Pull request by tdunkle**
### What does this Pull Request accomplish?

This pull request adds (almost) all of the functions, enums, and attributes from the NI-RFmx Bluetooth personality to grpc-device.
Fixes [AB#1784733](https://ni.visualstudio.com/DevCentral/_workitems/edit/1784733/)

### Why should this Pull Request be merged?

This pull request should be merged so that grpc-device can provide remote access to Bluetooth hardware via the NI-RFmx Bluetooth API.

### What testing has been done?

[AB#1811501](https://ni.visualstudio.com/DevCentral/_workitems/edit/1811501) was created to address an issue with the NI-RFmx Bluetooth functions 20dBBandwidthCfgAveraging, 20dBBandwidthFetchMeasurement, and 20dBBandwidthFetchSpectrum, whose presence broke the build of grpc-device since beginning the function name with a numeric rather than an alphabetic character is currently not supported. We will need to solve that problem in order to include those three functions.

Otherwise, building grpc-device succeeds locally when those three functions are excluded from the build.

### CODE REVIEW

**Comment by reckenro** [comment]
I think you're gonna have to update the system level tests for BT due to the updates in some of the parameter names for `CloseRequest` and `InitializeResponse`. Other than that, though, I think the changes look good.

**Review by reckenro — APPROVED** [review]
Looks good after you fix up and run the BT System-level tests.

File: generated/nirfmxbt/nirfmxbt.proto [hunk 1]
```diff
@@ -31,27 +325,1873 @@ enum Boolean {
...
+  NIRFMXBT_INT32_PAYLOAD_BIT_PATTERN_10101010 = 2;
+  NIRFMXBT_INT32_PAYLOAD_LENGTH_MODE_MANUAL = 0;
+  NIRFMXBT_INT32_PAYLOAD_LENGTH_MODE_AUTO = 1;
+  NIRFMXBT_INT32_TRIGGER_MINIMUM_QUIET_TIME_MODE_MANUAL = 0;
+  NIRFMXBT_INT32_TRIGGER_MINIMUM_QUIET_TIME_MODE_AUTO = 1;
+  NIRFMXBT_INT32_TRIGGER_TYPE_NONE = 0;
+  NIRFMXBT_INT32_TRIGGER_TYPE_DIGITAL_EDGE = 1;
+  NIRFMXBT_INT32_TRIGGER_TYPE_IQ_POWER_EDGE = 2;
+  NIRFMXBT_INT32_TRIGGER_TYPE_SOFTWARE = 3;
+  NIRFMXBT_INT32_TWENTYD_B_BANDWIDTH_AVERAGING_ENABLED_FALSE = 0;
```
Comment (astarche) [inline generated/nirfmxbt/nirfmxbt.proto:543]: Something to add to a future task (the API review story?) These names are messed up: TWENTYD_B.

File: generated/nirfmxbt/nirfmxbt.proto [hunk 2]
```diff
@@ -31,27 +325,1873 @@ enum Boolean {
...
+
+message ModAccFetchDf1Request {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  double timeout = 3;
+}
+
+message ModAccFetchDf1Response {
+  int32 status = 1;
+  double df1avg_maximum = 2;
```
Comment (astarche) [inline generated/nirfmxbt/nirfmxbt.proto:1663]: Not sure what these are supposed to be our what they started as. `df1_avg`? I guess the param names in the api metadata are `Df1avg` or something. I'd default to keeping it as-is, but we can see if there's any useful context in the .net or LV API.

File: generated/nirfmxbt/nirfmxbt.proto [hunk 3]
```diff
@@ -31,27 +325,1873 @@ enum Boolean {
...
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  double timeout = 3;
+}
+
+message TXPFetchEDRPowersResponse {
+  int32 status = 1;
+  double edrgfsk_average_power_mean = 2;
+  double edrdpsk_average_power_mean = 3;
+  double edr_d_psk_g_fsk_average_power_ratio_mean = 4;
```
Comment (astarche) [inline generated/nirfmxbt/nirfmxbt.proto:2111]: This is different than the 2 above it.

File: generated/nirfmxbt/nirfmxbt.proto (hunk 1 above)
Comment (astarche) [reply generated/nirfmxbt/nirfmxbt.proto:543]: ^^ Same goes for all of the proto comments. They just need a follow up task somewhere. Shouldn't block this PR or User Story.

**Review by astarche — APPROVED** [review]

**Review by mgumble — APPROVED** [review]

File: generated/nirfmxbt/nirfmxbt.proto (hunk 1 above)
Comment (tdunkle) [reply generated/nirfmxbt/nirfmxbt.proto:543]: Created [AB#1812664](https://ni.visualstudio.com/DevCentral/_workitems/edit/1812664).

File: generated/nirfmxbt/nirfmxbt.proto (hunk 2 above)
Comment (tdunkle) [reply generated/nirfmxbt/nirfmxbt.proto:1663]: Created [AB#1812677](https://ni.visualstudio.com/DevCentral/_workitems/edit/1812677).

File: generated/nirfmxbt/nirfmxbt.proto (hunk 3 above)
Comment (tdunkle) [reply generated/nirfmxbt/nirfmxbt.proto:2111]: Created [AB#1812687](https://ni.visualstudio.com/DevCentral/_workitems/edit/1812687).

**Comment by tdunkle** [comment]
There are several functions that begin with the word 'Build' that are breaking my ability to build grpc-device locally, and the same appears to be occurring in the desktop OS builds on this pull request. I am going to push a commit that removes these functions to see if I can get the build to succeed and submit the PR. Assuming that I can, I will create a follow-up task to get those functions fixed and included in the build before I submit this PR.

**Comment by astarche** [comment]
@tdunkle I should have been checking for these closer in the extraction changes. The BuildString methods are all (a) not present in the API metadata, (b) vary from personality-to-personality, and (c) are all basically the same params.

Those are addressed by adding them to the `handmade` metadata for the driver, and you will also want to remove all the ones that are (probabl)y copied from a different personality.

I'm also working on some of the diff logic to try to catch those better.

**Pull request merged**